src/golang/lib/databricks/jobs.go

```diff
@@ -95,21 +86,95 @@ func CreateJob(
 	return createResp.JobId, nil
 }
 
-func RunNow(
+func CreateTask(
 	ctx context.Context,
 	databricksClient *databricks_sdk.WorkspaceClient,
-	jobId int64,
+	workflowName string,
+	name string,
+	upstreamTaskNames []string,
+	pythonFilePath string,
 	specStr string,
+) (*jobs.JobTaskSettings, error) {
+
+	jobClusterKey := workflowNameToJobClusterKey(workflowName)
+
+	taskDependenciesList := make([]jobs.TaskDependenciesItem, 0, len(upstreamTaskNames))
+	for _, taskName := range upstreamTaskNames {
+		taskDependenciesList = append(taskDependenciesList, jobs.TaskDependenciesItem{TaskKey: taskName})
+	}
+
+	task := &jobs.JobTaskSettings{
+		TaskKey:       name,
+		JobClusterKey: jobClusterKey,
+		DependsOn:     taskDependenciesList,
+		SparkPythonTask: &jobs.SparkPythonTask{
+			PythonFile: pythonFilePath,
+			Parameters: []string{"--spec", specStr},
+		},
+		Libraries: []libraries.Library{
+			{
+				Pypi: &libraries.PythonPyPiLibrary{
+					Package: fmt.Sprintf("aqueduct-ml==%v", lib.ServerVersionNumber),
+				},
+			},
+			{
+				Pypi: &libraries.PythonPyPiLibrary{
+					Package: "snowflake-sqlalchemy",
+				},
+			},
```

Comment:
I have similar question if we should capture any requirement.txt dependencies here, or this can be done in follow-ups?

Reply:
requirements.txt dependencies are captured in the function execution, this is only for the python packages required for the data connector.